packages/samples/headless-react/src/components/smart-snippet/smart-snippet.class.tsx

```diff
@@ -50,13 +51,78 @@ export class SmartSnippet extends Component<{}, SmartSnippetState> {
       return null;
     }
 
-    const {answerFound, answer, question, liked, disliked, expanded} =
-      this.state;
+    const {
+      answerFound,
+      answer,
+      question,
+      liked,
+      disliked,
+      expanded,
+      feedbackModalOpened,
+    } = this.state;
 
     if (!answerFound) {
       return <div>Sorry, no answer has been found for this query.</div>;
     }
 
+    if (feedbackModalOpened) {
+      return (
+        <div role="dialog">
```

Comment:
how do we do the styling of these?

Reply:
we don't do styling on the `headless-react` package, it's purely meant to be code that is read by developers.